app/controllers/good_job/metrics_controller.rb

```diff
@@ -0,0 +1,16 @@
+module GoodJob
+  class MetricsController < ApplicationController
+    def primary_nav
+      jobs_count = number_to_human(GoodJob::Job.count)
+      batches_count = number_to_human(GoodJob::BatchRecord.migrated? ? GoodJob::BatchRecord.all.size : 0)
+      cron_entries_count = GoodJob::CronEntry.all.size
+      render json: { jobsCount: jobs_count, batchesCount: batches_count, cronEntriesCount: cron_entries_count }
```

Comment:
nice and simple 👍🏻 

This is a personal preference: I'd like these values to be snake_case instead of camelCase